### PR TITLE
milestone9: Authorization

### PR DESCRIPTION
**Pull request by abdessalem1998**
hello reviewer
In this pull request, I need to review my code for adding an Authorization for the Blog app. in this part we did the following :

- Install CanCanCan in your project.
- Add a `role` column to the users table. Remember to use a migration for this.
- A user can delete a post if it is theirs or if they have an admin role (column `role` has value `"admin"`). Use CanCanCan for this authorization.
    - For that you need to implement the post deleting functionality. Add the "Delete" button to the view and make sure that only authorized users can see it.
- A user can delete a comment if it is theirs or if they have an admin role (column `role` has value `"admin"`). Use CanCanCan for this authorization.
    - For that you need to implement the comment deleting functionality. Add the "Delete" button to the view and make sure that only authorized users can see it.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -9,6 +11,13 @@ class User < ApplicationRecord
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
+  # authorization Roles
+  Roles = %i[admin default].freeze
+
+  def is?(requested_role)
+    role == requested_role.to_s
+  end
```
Comment (VICTOR-CHUKWU) [inline app/models/user.rb:19]: - [ ] Nice Implementation @abdessalem1998  :clap:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @abdessalem1998 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (abdessalem1998) [reply app/models/user.rb:19]: thanks, dear

Comment (koachcode) [reply app/models/user.rb:19]: Thank you @VICTOR-CHUKWU

**Pull request merged**